packages/astro/astro.js

```diff
@@ -32,6 +32,13 @@ async function main() {
 		}
 	}
 
+  // windows drive letters can sometimes be lowercase, which vite cannot process
+  if (process.platform === 'win32') {
+    const cwd = process.cwd()
+    const correctedCwd = cwd.slice(0, 1).toUpperCase() + cwd.slice(1)
+    process.chdir(correctedCwd)
```

Comment:
Should we skip this it's already uppercase? And does this also fix the issue if the drive letter is originally lowercase?

Reply:
IIUC there could be two case:

1. Originally `C:/something/...` but the terminal is on `c:/something/...` (this is fixed)
2. Originally `c:/something/...` but the terminal is on `C:/something/...`

I'm not sure if no2 is a real-world scenario, but if so I was thinking the the `correctedCwd` would have it lowercase. But if no2 isn't something we should worry about, then the current code should be fine then.

Reply:
Yeah, while looking around github and discord threads, I did not come across a case where a lowercase drive letter is intentional on any software or user's part.